packages/nhsuk-frontend/src/nhsuk/core/styles/_typography.scss

```diff
@@ -149,6 +149,7 @@ h6,
 }
 
 p,
+.nhsuk-body,
```

Comment:
@colinrotherham The file upload component adds this class to a `span` not `p`, so without adding this alias, the status message does not appear with the correct font size. 

GOV.UK Frontend also adds this alias, but without getting into the weeds, slightly differently (but for which we can address in a separate PR if and when we want to align how we manage out typographic styles).